.github/workflows/update-loci.yaml

```diff
@@ -41,11 +44,31 @@ jobs:
           conda info
 
       - name: Check loci and update bed files (short)
-        #if: ${{ github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'}}
+        if: ${{ github.event_name == 'pull_request'}}
         run: snakemake --config stages="skip-refs"
 
+      # Enable this step to open an SSH session for debugging
+      # - name: SSH debug
+      #   uses: mxschmitt/action-tmate@v3
+
+      - name: Find new literature (intermediate)
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          conda init
+          source /home/runner/.bashrc
+          conda activate strchive
+          snakemake --config stages="new-refs"
+
+      - name: Update all literature (long)
+        if: ${{ github.event_name == 'schedule' }}
+        run: |
+          conda init
+          source /home/runner/.bashrc
+          conda activate strchive
+          snakemake
+
       - name: Open pull request with updated files
-        if: ${{ !(github.event_name == 'pull_request') }}
+        if: ${{ !(github.event_name == 'pull_request') || github.event_name == 'schedule' }}
```

Comment:
This "or schedule" is also unnecessary as it will already be covered by "not pull request"